workflow/rocoto_funcs/ungrib_lbc.py

```diff
@@ -8,16 +8,28 @@ def ungrib_lbc(xmlFile, expdir, do_ensemble=False):
   offset=int(os.getenv('LBC_OFFSET','6'))
   length=int(os.getenv('LBC_LENGTH','12'))
   interval=int(os.getenv('LBC_INTERVAL','3'))
+#  fhr=os.getenv('FCST_LENGTH','6')
```

Comment:
If we no longer use `fhr`, let's just remove line 11. We can always add it back if we have new needs.

Reply:
Meant to remove it after testing but forgot to do so. Thanks @guoqing-noaa